APMrover2/sailboat.cpp

```diff
@@ -90,25 +128,31 @@ Sailboat::Sailboat()
 void Sailboat::init()
 {
     // sailboat defaults
-    if (enabled()) {
+    if (sail_enabled()) {
         rover.g2.crash_angle.set_default(0);
+    }
+
+    if (nav_enabled()) {
         rover.g2.loit_type.set_default(1);
-        rover.g2.loit_radius.set_default(5);
-        rover.g2.wp_nav.set_default_overshoot(10);
+    }
+
+    // if we have a throttle of some sort allow to use it
```

Comment:
it's a crazy edge case but it could also theoretically be an omni sailboat.. insanely unlikely though..

Reply:
fixed, would love to see a onmi sailboat...